astro.config.ts

```diff
@@ -21,6 +53,59 @@ export default defineConfig({
         },
         plugins: [tailwindcss()]
     },
+
+    experimental: {
+        fonts: [
+            {
+                provider: 'local',
+                name: 'Source Sans Pro',
+                cssVariable: '--custom-font-sans',
+                fallbacks: FONTS_FALLBACKS.SANS,
```

Comment:
Simplify font fallbacks to rely on the browser’s default system font stack instead. Astro attempts to generate [optimized fallbacks](https://developer.chrome.com/blog/font-fallbacks) using font metrics.
```suggestion
                fallbacks: ['sans-serif'],
```